public/index.html

```diff
@@ -12,7 +12,7 @@
   <title>GHGA | The German Human Genome-Phenome Archive</title>
 </head>
 
-<body>
+<body style="overflow-y: visible">
   <noscript>You need to enable JavaScript to run this app.</noscript>
   <div id="root"></div>
 </body>
```

Comment:
There are some warnings and errors in the UI console, please fix them

Reply:
The warning still persists, can you please fix it? Upon loading of the page, you can see the warning on the browser console. `Warning: Each child in a list should have a unique "key" prop. and much more`